src/coffee/utils/array-sync.coffee

```diff
@@ -46,8 +46,7 @@ angular.module('uiGmapgoogle-maps').factory 'uiGmaparray-sync', [
           set_at: (index) ->
             return if isSetFromScope #important to avoid cyclic forever change loop watch to map event change and back
             value = mapArray.getAt(index)
-            return  unless value
-            return  if not value.lng or not value.lat
+            return unless value and value.lng and value.lat
```

Comment:
@nmccready, nice! isn't as readable though. plus it checks for the existence of `value` twice. to clarify:

original code:
return if `value` is undefined
return if `value` doesn't have a `lng` **or** it doesn't have a `ltd`

my code:
return unless there is a `value` defined with `lng` and `ltd`

your code:
return unless `value` is defined and has a `lng` **and** `value` is defined and has a `lat`

what do u say?